File: introduccion/clase2_condicionales.py
# Condicionales en Python

# if-else básico
edad = 20
# Esto es muy fácil: si se cumple la 'lógica', es decir, si la condición resulta True,
# entonces se ejecutará el código dentro del bloque 'if'. Si no se cumple, se pasará al 'else'.
if edad >= 18:
    print("Eres mayor de edad.")
else:
    print("Eres menor de edad.")

# if-elif-else
temperatura = 30

if temperatura > 30:
    print("Hace mucho calor.")
elif temperatura > 20:
    print("El clima es agradable.")
else:
    print("Hace frío.")

# Operador ternario (Python no tiene un operador ternario como tal, pero se puede hacer algo similar)
es_mayor_de_edad = "Sí, es mayor de edad." if edad >= 18 else "No, es menor de edad."
print(es_mayor_de_edad)

# Python no tieneswitch-case como javaScript

File: introduccion/clase2_condicionales.js
// Condicionales en JavaScript

let edad = 20;

// if-else básico
// Esto es muy fácil: si se cumple la 'lógica', es decir, si la condición resulta true, lo que esta entre "()",
// entonces se ejecutará el código dentro del bloque 'if'. Si no se cumple, se pasará al 'else'.
if (edad >= 18) {
    console.log("Eres mayor de edad.");
} else {
    console.log("Eres menor de edad.");
}

// if-else if-else
let temperatura = 30;

if (temperatura > 30) {
    console.log("Hace mucho calor.");
} else if (temperatura > 20) {
    console.log("El clima es agradable.");
} else {
    console.log("Hace frío.");
}

// Operador ternario (otra forma de hacerlo)
// si la condición resulta true, corre lo que va luego del "?" sino lo que va luego de ":" 
let es_mayor_de_edad = (edad >= 18) ? "Sí, es mayor de edad." : "No, es menor de edad.";
console.log(es_mayor_de_edad);

// switch-case
let dia = 3;
let nombreDia;

switch (dia) {
    case 1:
        nombreDia = "Lunes";
        break;
    case 2:
        nombreDia = "Martes";
        break;
    case 3:
        nombreDia = "Miércoles";
        break;
    case 4:
        nombreDia = "Jueves";
        break;
    case 5:
        nombreDia = "Viernes";
        break;
    case 6:
        nombreDia = "Sábado";
        break;
    case 7:
        nombreDia = "Domingo";
        break;
    default:
        nombreDia = "Día no válido";
        break;
}

console.log("Hoy es:", nombreDia);
